product.py

```diff
@@ -291,6 +291,19 @@ def fetchIfaceInfoFromNetworkdbAsDict(bridge, iface=None):
                     d['interfaces'] = p['interfaces']
                     d['hwaddrs'] = p.get('hwaddrs')
 
+                interface_rename_file = self.join_state_path(
+                    "etc/sysconfig/network-scripts/interface-rename-data/dynamic-rules.json"
```

Comment:
minor: better to define `interface_rename_dir` in wider scope and share with this one,  refer to `convert_interface_rename_data`